CMakeLists.txt

```diff
@@ -180,3 +180,19 @@ add_custom_target(
   COMMAND ${CMAKE_SOURCE_DIR}/scripts/bench.sh
   WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
  )
+
+# Package universal install script
+add_custom_target(
+  archive
+  ${CMAKE_COMMAND} -E env SOURCE_DIR=${CMAKE_SOURCE_DIR} BUILD_DIR=${CMAKE_BINARY_DIR} ${CMAKE_SOURCE_DIR}/scripts/package.sh
+  DEPENDS ${CMAKE_BINARY_DIR}/lanterndb.so
```

Comment:
This depends on the whole target (the control file, the generated SQL files) and not just the shared library, right?
I think you can do `add_dependency(archive lanterndb)` to better capture this.
I am just afraid that if we do something that only modifies an SQL file, and already had an old build, `make archive` will not re-trigger a build.